platform-emf/src/main/java/com/softicar/platform/emf/attribute/AbstractEmfAttribute.java

```diff
@@ -153,6 +148,37 @@ public final boolean isMandatory(R tableRow) {
 		return this;
 	}
 
+	public final AbstractEmfAttribute<R, V> setValueDeducer(Consumer<R> valueDeducer) {
+
+		this.valueDeducer = valueDeducer;
+		return this;
+	}
+
+	public final AbstractEmfAttribute<R, V> setValueDeducer(IEmfPredicate<R> predicate, V value) {
+
+		return setValueDeducer(row -> {
+			if (predicate.test(row)) {
+				setValue(row, value);
+			}
+		});
+	}
+
+	// ------------------------------ Predicate Convenience ------------------------------ //
+
+	public AbstractEmfAttribute<R, V> setConditionallyAvailable(IEmfPredicate<R> predicate, V fallbackValue) {
```

Comment:
- I'd personally find `setDynamicallyAvailable` more intuitive.
- This method should have some Javadoc.

Reply:
hmm, I would rather prefer the word `Conditionally`